test/manifold_test.cpp

```diff
@@ -833,3 +834,165 @@ TEST(Manifold, EmptyHull) {
       {0, 0, 0}, {1, 0, 0}, {0, 1, 0}, {1, 1, 0}};
   EXPECT_TRUE(Manifold::Hull(coplanar).IsEmpty());
 }
+
+TEST(Manifold, MinGapCubeCube) {
+  auto a = Manifold::Cube();
+  auto b = Manifold::Cube().Translate({2, 2, 0});
+
+  float distance = a.MinGap(b, 1.5f);
+
+  EXPECT_FLOAT_EQ(distance, sqrt(2));
+}
+
+TEST(Manifold, MinGapCubeCube2) {
+  auto a = Manifold::Cube();
+  auto b = Manifold::Cube().Translate({3, 3, 0});
+
+  float distance = a.MinGap(b, 3);
+
+  EXPECT_FLOAT_EQ(distance, sqrt(2) * 2);
+}
+
+TEST(Manifold, MinGapCubeCubeOutOfBounds) {
+  auto a = Manifold::Cube();
+  auto b = Manifold::Cube().Translate({3, 3, 0});
+
+  float distance = a.MinGap(b, 2.5f);
+
+  EXPECT_FLOAT_EQ(distance, 2.5f);
+}
+
+TEST(Manifold, MinGapCubeSphereOverlapping) {
+  auto a = Manifold::Cube();
+  auto b = Manifold::Sphere(1);
+
+  float distance = a.MinGap(b, 0.1f);
+
+  EXPECT_FLOAT_EQ(distance, 0);
+}
+
+TEST(Manifold, MinGapSphereSphere) {
+  auto a = Manifold::Sphere(1);
+  auto b = Manifold::Sphere(1).Translate({2, 2, 0});
+
+  float distance = a.MinGap(b, 0.85f);
+
+  EXPECT_FLOAT_EQ(distance, 2 * sqrt(2) - 2);
+}
+
+TEST(Manifold, MinGapSphereSphere2) {
+  auto a = Manifold::Sphere(1);
+  auto b = Manifold::Sphere(1).Translate({2, 2, 0});
+
+  float distance = a.MinGap(b, 0.85f);
+
+  EXPECT_FLOAT_EQ(distance, 2 * sqrt(2) - 2);
```

Comment:
Is this test identical to the one above?

Reply:
Yep, removed it